app/controllers/formbuilder/edit/configuration.js

```diff
@@ -152,6 +157,33 @@ export default class FormbuilderConfigurationController extends Controller {
     return JSON.parse(option.value).conceptScheme ?? null;
   }
 
+  get sectionHasOneTableListing() {
+    const tableListings = this.builderStore
+      .match(
+        undefined,
+        RDF('type'),
+        FORM('ListingTable'),
+        this.model.graphs.sourceGraph
+      )
+      .map((triple) => triple.subject);
+
+    if (!tableListings || tableListings.length == 0) return false;
+
+    for (const tableListing of tableListings) {
+      const isPartOf = this.builderStore.any(
+        tableListing,
+        FORM('partOf'),
+        this.selectedSection.parent
+      );
+
+      if (!isPartOf) continue;
+
+      return true;
+    }
+
+    return false;
```

Comment:
Should we just pass the table listing of the selected section to our component?

Reply:
Yes we should as if we do it like this we are adding the limits to all the tables in the selected section